src/EA.Weee.Web.Tests.Unit/ViewModels/EvidenceNoteRowViewModelTests.cs

```diff
@@ -127,16 +127,16 @@ public void AatfViewRouteName_GivenRejectedNoteStatus_RejectedViewShouldBeReturn
         }
 
         [Fact]
-        public void AatfViewRouteName_GivenStatusHasNoRoute_InvalidOperationExceptionExpected()
+        public void AatfViewRouteName_GivenVoidNoteStatus_VoidedViewShouldBeReturned()
         {
             //arrange
             model.Status = NoteStatus.Void;
 
             //act
-            var exception = Record.Exception(() => model.AatfViewRouteName);
+            var result = model.AatfViewRouteName;
 
             //assert
```

Comment:
@GuidoWorkDev  Just noticed, is this correct? The AATF AatfViewRouteName returns a scheme named route SchemeTransferEvidenceRedirect.ViewVoidedTransferEvidenceRouteName?

Reply:
added some code to resolve AATF view void note